BookApi/BookApi/Program.cs

```diff
@@ -0,0 +1,32 @@
+using BookApi;
+using BookApi.Service;
+using BookApi.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
```

Comment:
```suggestion
using Microsoft.EntityFrameworkCore;
```

VSCode shows these as unused imports :)